### PR TITLE
Class renamings, mainly entity-related

### PR DESCRIPTION
**Pull request by YanisBft**
Renamed classes to match ingame entity IDs
Also sorted the classes that were in net/minecraft/client/sortme

### CODE REVIEW

**Comment by liach** [comment]
btw, did git rename tnt minecart mapping file name after you changed case?

**Comment by kashike** [comment]
> btw, did git rename tnt minecart mapping file name after you changed case?

Nope. The file needs renaming.

File: mappings/net/minecraft/entity/vehicle/TNTMinecartEntity.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS axv net/minecraft/entity/vehicle/TNTMinecartEntity
```
Comment (liach) [inline mappings/net/minecraft/entity/vehicle/TNTMinecartEntity.mapping:1]: Do `git mv mappings/net/minecraft/entity/vehicle/TNTMinecartEntity.mapping mappings/net/minecraft/entity/vehicle/TntMinecartEntity.mapping`

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/vehicle/TNTMinecartEntity.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/entity/vehicle/TNTMinecartEntity.mapping:1]: Thanks, renamed

**Review by modmuss50 — APPROVED** [review]
Seems easy enough

**Review by liach — APPROVED** [review]
Modmuss please merge!

**Pull request merged**